src/lib/runner.rs

```diff
@@ -38,7 +38,7 @@ where
             if cwd.len() > 0 {
                 let directory = envmnt::get_or("CARGO_MAKE_WORKING_DIRECTORY", "");
 
-                environment::setup_cwd(Some(cwd));
+                environment::setup_cwd(Some(&super::environment::expand_value(cwd)));
```

Comment:
it may expand to an empty string so i think another len>0 is needed.
naturally do it before line 39.
also, why super?

Reply:
> also, why super?

Im baffled myself 😕.